src/components/BookCardsPage/BookCardsPage.js

```diff
@@ -76,12 +84,12 @@ const CardsPageLayout = styled.div`
   position: relative;
   padding: 2%;
   &:first-child {
-    padding-top: 55px;
+    padding-top: 105px;
   }
 `
-const ButtonWrapper = styled.div`
-  justify-content: space-around;
-  display: flex;
+const TabBarWrapper = styled.div`
+  display: grid;
+  grid-template-columns: 50% 50%;
```

Comment:
Just an alternative way of writing this, but maybe you'd like to use fractions here instead? If you ever wanted a third or fourth row (perhaps on bigger screens?) you wouldn't have to manually adjust percentages, if you wrote `grid-template-columns: 1fr 1fr`.